app/src/test/java/rick_and_morty/data/repository/CharacterRepositoryTest.kt

```diff
@@ -1,5 +1,26 @@
 package rick_and_morty.data.repository
 
-import org.junit.Assert.*
+import com.google.common.truth.Truth.assertThat
+import com.nhaarman.mockitokotlin2.*
+import kotlinx.coroutines.runBlocking
+import org.junit.Test
+import rick_and_morty.data.model.RickAndMortyResponseDto
+import rick_and_morty.di.RickAndMortyApiRemoteDataSource
 
-class CharacterRepositoryTest
\ No newline at end of file
+class CharacterRepositoryTest {
+
+    private val rickAndMortyApiRemoteDataSource: RickAndMortyApiRemoteDataSource = mock ()
+    private val classToTest = CharacterRepository(rickAndMortyApiRemoteDataSource)
+    private val rickAndMorty: RickAndMortyResponseDto = mock ()
+
+    @Test
+    fun `verify getCharacter has been called`(): Unit = runBlocking {
+
+        given(rickAndMortyApiRemoteDataSource.fetchRickAndMortyData(1)).willReturn(rickAndMorty)
+
+        classToTest.getCharacters(1)
+
+        verify(rickAndMortyApiRemoteDataSource).fetchRickAndMortyData(1)
+        assertThat(rickAndMortyApiRemoteDataSource.fetchRickAndMortyData(1).results).isEmpty()
```

Comment:
`CharacterRepository` is tested in this class, so no results coming from functions of any other class (like `RickAndMortyApiRemoteDataSource`) should be checked here.

Reply:
Done